backend/apis/normal-docket-pdf.test.js

```diff
@@ -29,3 +31,52 @@ describe("normal docket pdf file", () => {
     expect(actual1.dispositionDate).toBe("Month 00, 0000");
   });
 });
+
+describe("correctly and completely parses ACCOUNT SUMMARY section", () => {
```

Comment:
Let's create two test suites (`describe()` calls) - one for each test file. The variable names inside of this test suite (`parsedSection`, for example) make it hard to understand which test file is being tested with each assertion.

Good job writing these tests, though. They are validating that the correct things are being extracted 👍

Reply:
I'll split them up.

Reply:
About the tests, I'm using the test example Veronica included as an example of a docket with no content in the Account Summary section. One of my test files is meant to see if the function can parse a large files traversing several pages, and the other to see if it can get the Debt Collection case right. The describe blocks are testing multiple files because they represent different edge case. 

That said, should I repeat the tests with nested describe blocks, like this:
describe("testing file A", ()=>{
   describe("testing edge case 1", ()=>{}
})

or should I just rename the files and variables so that they make more sense?

Reply:
You should have two `describe()` blocks: one for each test.